docs_user/modules/proc_migrating-mon-from-controller-nodes-network.adoc

```diff
@@ -11,12 +11,19 @@ IP address migration assumes that the target nodes are originally deployed by
 // w/ an EDPM node that has already been adopted.
 .Procedure
 
-. Get the original Ceph Monitor IP address from the existing `/etc/ceph/ceph.conf` file on the `mon_host` line, for example:
+. Get the original Ceph Monitor IP addresses from `$HOME/ceph_client_backup/ceph.conf` file on the `mon_host` line, for example:
 +
 ----
 mon_host = [v2:172.17.3.60:3300/0,v1:172.17.3.60:6789/0] [v2:172.17.3.29:3300/0,v1:172.17.3.29:6789/0] [v2:172.17.3.53:3300/0,v1:172.17.3.53:6789/0]
 ----
 
+. Match it with storage newtork IP addresses on the source node and find the Ceph Monitor IP address
```

Comment:
instead if `it`, we should explicitly say: `the ip address retrieved in the previous step`. @klgill to give the right direction here.

Reply:
done, @klgill can review the same and edit the statement if required.